chain/jsonrpc/src/lib.rs

```diff
@@ -411,6 +411,9 @@ impl JsonRpcHandler {
             "EXPERIMENTAL_changes_in_block" => {
                 process_method_call(request, |params| self.changes_in_block(params)).await
             }
+            "EXPERIMENTAL_congestion_info" => {
+                process_method_call(request, |params| self.congestion_info(params)).await
+            }
```

Comment:
I think it would be better to call it `EXPERIMENTAL_congestion_level` since congestion_info has a different meaning.

Reply:
Huh, how did I not notice this :sweat_smile: The PR name even has the name correct but somehow here it's wrong^^ Thanks for catching!